components/byline/class-byline.php

```diff
@@ -12,10 +12,93 @@
  */
 class Byline extends Component {
 
+	use \WP_Component\WP_User;
+	use \WP_Component\Guest_Author;
+
 	/**
 	 * Unique component slug.
 	 *
 	 * @var string
 	 */
 	public $name = 'byline';
+
+
+	/**
+	 * Define a default config.
+	 *
+	 * @return array Default config.
+	 */
+	public function default_config() {
+		return [
+			'link' => '',
+			'name' => '',
+		];
+	}
+
+	/**
+	 * Get an array of byline components for a given post.
+	 *
+	 * @param int|null|\WP_Post $post_id Post ID, null to use global $post, or
+	 *                                   WP_Post object.
+	 * @return array Byline components.
+	 */
+	public static function get_post_bylines( $post_id = null ) {
```

Comment:
It might be useful to have an optional second param where you can override whether to use WP Users or coauthors, since there could be cases where the CAP plugin is active but not desired for bylines.

Reply:
I need to go back and update this to use the new Author trait. It'll take care of that problem.